sys/kern/subr_bus.c

```diff
@@ -472,14 +472,28 @@ driver_register_pass(struct driverlink *new)
 	TAILQ_INSERT_TAIL(&passes, new, passlink);
 }
 
+/**
+ * @brief Retrieve the current bus pass
+ *
+ * Retrieves the current bus pass level.  Call the BUS_NEW_PASS()
+ * method on the root bus to kick off a new device tree scan for each
+ * new pass level that has at least one driver.
+ */
+int
+bus_get_pass(void)
+{
+
+	return (bus_current_pass);
+}
+
 /**
  * @brief Raise the current bus pass
  *
  * Raise the current bus pass level to @p pass.  Call the BUS_NEW_PASS()
  * method on the root bus to kick off a new device tree scan for each
  * new pass level that has at least one driver.
  */
-void
+static void
```

Comment:
This whole change looks good, I'll pick it up. However, we may need to make this non-static

Reply:
Okay.  I really hope out-of-tree architectures have moved away from this, but that wasn't something I was aware of.  It looked effectively deprecated since there weren't any other uses.